mistralrs-server/src/main.rs

```diff
@@ -269,15 +269,7 @@ async fn main() -> Result<()> {
     if use_flash_attn {
         info!("Using flash attention.");
     }
-    if use_flash_attn
-        && matches!(
-            loader.get_kind(),
-            ModelKind::QuantizedGGML
-                | ModelKind::QuantizedGGUF
-                | ModelKind::XLoraGGML
-                | ModelKind::XLoraGGUF
-        )
-    {
+    if use_flash_attn && !loader.get_kind().is_quantized() {
```

Comment:
```suggestion
    if use_flash_attn && loader.get_kind().is_quantized() {
```
Same as above, I think the logic should be inverted.